src/ipc/handler.c

```diff
@@ -184,28 +183,25 @@ static int ipc_stream_pcm_params(uint32_t stream)
 	struct comp_dev *cd;
 	int err, posn_offset;
 
-	trace_ipc("ipc_stream_pcm_params()");
+	trace_ipc("ipc: comp %d -> params", pcm_params->comp_id);
 
 	/* sanity check size */
 	if (IPC_INVALID_SIZE(pcm_params)) {
-		trace_ipc_error("ipc_stream_pcm_params() error: Invalid size of"
-				"IPC pcm params.");
+		trace_ipc_error("ipc:_invalid IPC size 0x%x got 0x%x",
+				sizeof(*pcm_params), pcm_params->hdr.size);
 		return -EINVAL;
 	}
 
 	/* get the pcm_dev */
 	pcm_dev = ipc_get_comp(_ipc, pcm_params->comp_id);
 	if (pcm_dev == NULL) {
-		trace_ipc_error("ipc_stream_pcm_params() error: Requested "
-				"pcm_dev ID = %u not found.",
-				pcm_params->comp_id);
+		trace_ipc_error("ipc: comp %d not found", pcm_params->comp_id);
 		return -ENODEV;
```

Comment:
nit-pick: you've replaced %u by %d in the format, is this intentional?

Reply:
Will fix incrementally.